docker/ml-testing-toolkit/spec_files/api_definitions/mojaloop_sdk_outbound_scheme_adapter_1.0/api_spec.yaml

```diff
@@ -824,7 +824,7 @@ components:
       title: CorrelationId
       type: string
       pattern: >-
-        ^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$
+        ^[0-9a-f]{8}-[0-9a-f]{4}-[1-7][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$|^[0-9A-HJKMNP-TV-Z]{26}$
```

Comment:
pattern change but no change to the version number? is a version change involved?

Reply:
Don't quote me on this but I believe it was agreed that correlationId would be updated to handle ULID's in all specs retroactively. So no, no version change.

Reply:
:-) ok, not going to hold it for the sdk API but not for the FSPIOP API..